recipes/pycoal/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "pycoal" %}
+{% set version = "0.5.1" %}
+{% set sha256 = "d066af5e92d1d8250d972c7a683f70403df40e2aa3bac2d45655b301df7c7a18" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - numpy
+    - spectral
+
+test:
+  imports:
+    - pycoal
+
+about:
+  home: https://github.com/capstone-coal/pycoal/
+  license: Apache 2.0
+  license_family: Apache
+  license_file: LICENSE
```

Comment:
Thanks for catching that @jakirkham and @pmlandwehr!

Linking against a GPLv2 library might be construed to be acceptable under some circumstances, however the current version of COAL fundamentally depends on Spectral Python. In addition, it [appears](https://www.gnu.org/licenses/license-list.en.html#apache2) that GPLv2 and Apache2 are incompatible:

> Apache License, Version 2.0 (#apache2)
> 
> This is a free software license, compatible with version 3 of the GNU GPL.
> 
> Please note that this license is not compatible with GPL version 2, because it has some requirements that are not in that GPL version. These include certain patent termination and indemnification provisions. The patent termination provision is a good thing, which is why we recommend the Apache 2.0 license for substantial programs over other lax permissive licenses.

Possible actions:
1. Do nothing if the current licensing can be justified. Seek feedback from @tboggs or legal.
2. Upgrade to GPL with approval of @lewismc, @browtayl, @heidiaclayton, and @xiaomei7:
    * Version 2: Use the same license as Spectral Python.
    * Version 3: Use the updated license with stronger terms.
3. Implement the functionality from scratch or with a different library.
4. License COAL files (e.g., docs and source) under Apache v2 or GPL separately as with a [Debian copyright](https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/) file.

If (1) is not possible, then I recommend we relicense COAL under GPLv3.

For reference, GDAL uses an [X11/MIT license](https://trac.osgeo.org/gdal/wiki/FAQGeneral#WhatlicensedoesGDALOGRuse) and NumPy uses a [three-clause BSD license](https://docs.scipy.org/doc/numpy/license.html), in other words permissive in both cases.

Note that this should not affect the [website](https://github.com/capstone-coal/capstone-coal.github.io) or the [SDS](https://github.com/capstone-coal/coal-sds).